src/main/java/org/cryptomator/linux/keychain/KDEWalletKeychainAccess.java

```diff
@@ -24,7 +24,15 @@
 	public KDEWalletKeychainAccess() {
 		ConnectedWallet wallet = null;
 		try {
-			DBusConnection conn = DBusConnection.getConnection(DBusConnection.DBusBusType.SESSION);
+			DBusConnection conn = null;
+			try {
+				conn = DBusConnection.getConnection(DBusConnection.DBusBusType.SESSION);
+			} catch (RuntimeException e) {
```

Comment:
`DBusConnection.getConnection` throws a `DBusException` doesn't it? I'd suggest to catch that instead.

Reply:
It can throw a DBusException on certain errors but in the case of a non-existent session DBus it is only throwing a RuntimeException as per this line
`Caused by: java.lang.RuntimeException: Cannot Resolve Session Bus Address`
Using a DBusException in place of it results in the same error as before and the catch is never executed.
There may be a better way to check before the exception is thrown... Let me look into it a bit more.